tests/rgen/Microsoft.Macios.Bindings.Analyzer.Tests/BindingTypeSemanticAnalyzerTests.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Xamarin.Utils;
+using Xunit;
+
+namespace Microsoft.Macios.Bindings.Analyzer.Tests;
+
+public class BindingTypeSemanticAnalyzerTests : BaseGeneratorWithAnalyzerTestClass {
+
+	[Theory]
+	[InlineData (ApplePlatform.iOS)]
+	[InlineData (ApplePlatform.TVOS)]
+	[InlineData (ApplePlatform.MacOSX)]
+	[InlineData (ApplePlatform.MacCatalyst)]
+	public async Task BindingTypeMustBePartial (ApplePlatform platform)
```

Comment:
It doesn't look like you ignore the test when a platform is not enabled (by calling `Configuration.IgnoreIfIgnoredPlatform (platform)` somewhere).

Reply:
I can add a verification for the tests. Let me see if there is a way we can add that to xunit in a simple manner.